features/step_definitions/content.rb

```diff
@@ -26,6 +26,10 @@
   expect(page).to have_field(name, with: text)
 end
 
+Then /^there should be a dropdown '(.*)' with options '(.*)'$/ do |name, options|
+  expect(page).to have_select(name)#, with_options: options.split(','))
```

Comment:
dead code, and a severe case of it. Either implement the step as one would expect it to work, or re-phrase the step.